extensions/2.0/Khronos/KHR_materials_transmission/README.md

```diff
@@ -0,0 +1,138 @@
+# KHR\_materials\_transmission
+
+## Contributors
+
+* Mike Bond, Adobe, [@miibond](https://github.com/MiiBond)
+
+## Status
+
+Work in Progress
+
+## DependenciesI
```

Comment:
`DependenciesI` -> `Dependencies`

Reply:
Thanks. Finally addressed this (only 3 months late).